pkg/proxy/proxy.go

```diff
@@ -259,14 +266,14 @@ func (p *Proxy) ackProxyPort(pp *ProxyPort) error {
 
 		// Remove old rules, if any and for different port
 		if pp.rulesPort != 0 && pp.rulesPort != pp.proxyPort {
-			scopedLog.Debugf("Removing old proxy port rules for %s:%d", pp.name, pp.rulesPort)
+			scopedLog.Infof("Removing old proxy port rules for %s:%d", pp.name, pp.rulesPort)
```

Comment:
Are we planing to keep this as Info?

Reply:
I'd like to, for a while at least. It should not be chatty at all.